src/main/java/com/facebook/openwifirrm/modules/ApiServer.java

```diff
@@ -372,6 +416,22 @@ private void afterFilter(Request request, Response response) {
 		}
 	}
 
+	/** Global OPTIONS handler. */
+	private String options(Request request, Response response) {
+		// Set CORS headers
+		String acrh = request.headers("Access-Control-Request-Headers");
+		if (acrh != null) {
+			response.header("Access-Control-Allow-Headers", acrh);
+		}
+		String acrm = request.headers("Access-Control-Request-Method");
+		if (acrm != null) {
+			response.header("Access-Control-Allow-Methods", acrm);
+		}
+		response.header("Access-Control-Allow-Credentials", "true");
+		response.header("Access-Control-Max-Age", "86400");
```

Comment:
Maybe make 86400 a parameter

Reply:
I decided not to since this isn't important to change, it's just the cache time